octobot/backtesting/independent_backtesting.py

```diff
@@ -65,6 +65,10 @@ def __init__(self, config,
         self.optimizer_id = None
         self.backtesting_id = None
         self._init_default_config_values()
+        try:
+            self.backtesting_config["exchange_type"] = config["exchange_type"]
```

Comment:
Could we use a constant here (like `CONFIG_EXCHANGE_TYPE`) please ?

Reply:
thats already done